interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -16,6 +16,52 @@
 require_once("../../globals.php");
 require_once($GLOBALS['srcdir']."/options.inc.php");
 
+/**
+ * @param $wordInLang
+ * @return bool
+ */
+function ltox($wordInLang)
```

Comment:
What is this function now meant to do now?
(still trying to do reverse translation or given up on that?)

Reply:
Made a new SQL query to list option and was changed a function name to get_sex_word.